modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCacheUtils.java

```diff
@@ -1108,6 +1111,14 @@ public static int cacheGroupId(String cacheName, @Nullable String grpName) {
         return grpName != null ? CU.cacheId(grpName) : CU.cacheId(cacheName);
     }
 
+    /**
+     * @param ccfg Cache configuration.
+     * @return Group name if it is specified, otherwise cache name.
+     */
+    public static String cacheOrGroupName(CacheConfiguration<?, ?> ccfg) {
```

Comment:
I don't see any reason for duplicating this method. We already have CacheGroupContext.cacheOrGroupName. We can either extract this from CacheGroupContext and move it to GridCacheUtils, or just use the existing one.

Reply:
hanks. I removed it.